src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/Monitor.java

```diff
@@ -154,11 +155,10 @@ ConnectionStatus checkConnectionStatus(final int shortestFailureDetectionInterva
       if (this.monitoringConn == null || this.monitoringConn.isClosed()) {
         // open a new connection
         Map<String, String> properties = new HashMap<>();
```

Comment:
A bit confusing with properties and props.

I suggest calling this map "propertyMap"

Reply:
Renamed it to monitoringConnProperties. I hope it's clearer.